.github/pull_request_template.md

```diff
@@ -32,6 +32,10 @@ Closes #
 
 <!-- Describe any specific details to help reviewers test or review this Pull Request -->
 
+- [ ] Integration tests are run at github/github (Optional and only available for GitHub staff)
```

Comment:
Maybe we could mention that integration tests should pass?

Reply:
Ah yes, we should! I'll update it now.